g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/translation/dao/SingleComponentDao.java

```diff
@@ -30,6 +31,8 @@
 	 */
 	public ComponentMessagesDTO getTranslationFromFile(ComponentMessagesDTO componentMessagesDTO) throws L10nAPIException;
 
+	void saveCreationInfo(UpdateTranslationDTO updateTranslationDTO);
```

Comment:
Recommend add "public" key word on method in this interface

Reply:
done

Reply:
Can't change because test coverage will be less than 60%.